pkg/http/http.go

```diff
@@ -45,7 +45,7 @@ func NewClient(rl *rate.Limiter) *RLHTTPClient {
 // sha256 hash of it.
 //
 // On success, it will return the sha256 hash as a string.
-func (r *RLHTTPClient) GetArtifactSHA256(ctx context.Context, artifactURI string) (string, error) {
+func (c *RLHTTPClient) GetArtifactSHA256(ctx context.Context, artifactURI string) (string, error) {
```

Comment:
Why?

Reply:
Because in line 22 you have:
```go
func (c *RLHTTPClient) Do(req *http.Request) (*http.Response, error) {
```

`golint` flagged it with: `receiver name r should be consistent with previous receiver name c for RLHTTPClient`, in this case, I don't think it's completely irrelevant. IMHO, consistency can only help. 
While we're looking at this one though, not sure why the receiver is needed here but I could be missing something....

Reply:
Ok.